kms/api/Kms.Samples/SignAsymmetric.cs

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START kms_sign_asymmetric]
+
+using Google.Cloud.Kms.V1;
+using Google.Protobuf;
+using System.Security.Cryptography;
+using System.Text;
+
+public class SignAsymmetricSample
+{
+    public byte[] SignAsymmetric(
+      string projectId = "my-project", string locationId = "us-east1", string keyRingId = "my-key-ring", string keyId = "my-key", string keyVersionId = "123",
+      string message = "Sample message")
+    {
+        // Create the client.
+        KeyManagementServiceClient client = KeyManagementServiceClient.Create();
+
+        // Calculate the message digest.
+        SHA256 sha256 = SHA256.Create();
+        byte[] digest = sha256.ComputeHash(Encoding.UTF8.GetBytes(message));
```

Comment:
Let's convert the text to UTF-8 bytes earlier and separately.  Same family of concerns as previous comments about making it very clear that our interfaces are designed to accept binary data.

Reply:
Done